### DISCUSSION TITLE
Blur Detection

### DISCUSSION THREAD

**Discussion by uttu-316** [oliver-moran/jimp#1230] [Q&A]
Is there any way to detect the blurriness of the image? 

Blurness can be motion blur, radial blur, and overall blur.

Also, is there any way to get a sharpness score to detect the brightness, and darkness of the image?

I need to perform an image analysis.

**Comment by hipstersmoothie** [comment]
jimp itself gives you the bitmap but you would have to do the image analysis yourself
